src/Pokemon/SmallView/smallListing.js

```diff
@@ -8,7 +8,7 @@ import { useNavigate } from "react-router";
 const ProfileListing = ({
```

Comment:
some of the static sprites are clipping out of the box but that's out of the scope of this ticket probably